daemon/config.go

```diff
@@ -80,6 +80,7 @@ func (config *Config) InstallFlags() {
 	opts.UlimitMapVar(config.Ulimits, []string{"-default-ulimit"}, "Set default ulimits for containers")
 	flag.StringVar(&config.LogConfig.Type, []string{"-log-driver"}, "json-file", "Default driver for container logs")
 	flag.BoolVar(&config.Bridge.EnableUserlandProxy, []string{"-userland-proxy"}, true, "Use userland proxy for loopback traffic")
+	opts.LogOptsVar(config.LogConfig.Config, []string{"-log-opt"}, "Set log driver options")
```

Comment:
@wlan0 isn't this supposed to be `&config.LogConfig.Config`  (not sure, just reading the code)